lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -202,8 +203,18 @@ function REPL( options ) {
 	// Define the current workspace:
 	setNonEnumerable( this, '_currentWorkspace', 'base' );
 
-	// Initialize an internal flag indicating whether a user is entering a multi-line command:
-	setNonEnumerable( this, '_multiline_mode', false );
+	// Initialize an internal object storing user settings:
+	setNonEnumerable( this, '_settings', {} );
+	setNonEnumerable( this._settings, 'autoCloseSymbols', true );
+
+	// Initialize an internal status object for controlling multi-line mode:
+	setNonEnumerable( this, '_multiline_mode', {} );
+	setNonEnumerable( this._multiline_mode, 'status', false );
```

Comment:
```suggestion
	setNonEnumerable( this._multiline_mode, 'active', false );
```

I wouldn't expect a "status" property or variable to have a boolean value a priori, so I'd change to something which conveys better the value type. Could also do something like `isEnabled` or `isActive`.